### PR TITLE
Display failed tests when StopOnFailedTest is false

### LINKED ISSUE #355: Display failed tests
Currently, if `stopOnFailedTest` is set to `false`, Medusa keeps running indefinitely. The issue is that it does not show how many tests have currently failed, which could be helpful in determining when to stop the fuzzer.

It would be nice to have some statistics similar to that of Echidna, which shows `failed: N/M` on output logs

### PR DESCRIPTION
**Pull request by antks03**
close #355

### CODE REVIEW

File: fuzzing/fuzzer.go [hunk 1]
```diff
@@ -16,6 +15,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/crytic/medusa/fuzzing/executiontracer"
```
[the remark this answers is not in the document]
Comment (antks03) [reply fuzzing/fuzzer.go:18]: Probably it was a automatic change and I didn't see.

File: fuzzing/fuzzer.go [hunk 2]
```diff
@@ -86,6 +87,10 @@ type Fuzzer struct {
 
 	// logger describes the Fuzzer's log object that can be used to log important events
 	logger *logging.Logger
+
+	// Counts how many tests passed or not during the fuzzing test
+	numTestsPassed int
+	numTestsFailed int
```
[the remark this answers is not in the document]
Comment (antks03) [reply fuzzing/fuzzer.go:93]: Will do it

File: fuzzing/fuzzer.go [hunk 3]
```diff
@@ -787,13 +802,17 @@ func (f *Fuzzer) printMetricsLoop() {
...
+		totalTests := f.numTestsPassed + f.numTestsFailed
+
 		// Print a metrics update
 		logBuffer := logging.NewLogBuffer()
 		logBuffer.Append(colors.Bold, "fuzz: ", colors.Reset)
 		logBuffer.Append("elapsed: ", colors.Bold, time.Since(startTime).Round(time.Second).String(), colors.Reset)
 		logBuffer.Append(", calls: ", colors.Bold, fmt.Sprintf("%d (%d/sec)", callsTested, uint64(float64(new(big.Int).Sub(callsTested, lastCallsTested).Uint64())/secondsSinceLastUpdate)), colors.Reset)
 		logBuffer.Append(", seq/s: ", colors.Bold, fmt.Sprintf("%d", uint64(float64(new(big.Int).Sub(sequencesTested, lastSequencesTested).Uint64())/secondsSinceLastUpdate)), colors.Reset)
 		logBuffer.Append(", coverage: ", colors.Bold, fmt.Sprintf("%d", f.corpus.ActiveMutableSequenceCount()), colors.Reset)
+		logBuffer.Append(", failed: ", colors.Bold, fmt.Sprintf("%d/%d", f.numTestsFailed, totalTests), colors.Reset)
```
Comment (0xalpharush) [inline fuzzing/fuzzer.go:815]: I don't think it's necessary to track the `numTestsPassed` and sum the total. Instead `sequencesTested` can be used as the total.

**Comment by 0xalpharush** [comment]
It may be helpful to also track the number of reverting transactions out of the total number executed as that may highlight an issue

Also, can you sign the CLA please?

**Comment by 0xalpharush** [comment]
Any help needed to get this over the line?

**Comment by 0xalpharush** [comment]
This was merged and you'll see `failures: X/num. total call sequences `  now

**Pull request closed without merging**